src/pages/Product/Product.layout.vue

```diff
@@ -29,7 +29,7 @@
 
 <script lang="ts" setup>
 import DetailLayout from 'src/components/DetailLayout';
-import DIDLogo from 'src/assets/logos/didperu.svg';
```

Comment:
could you check if the old image is still in use? if not, delete it.

Reply:
@Eghinner Yes it is still used

Reply:
ok